extensions/undertow/deployment/src/test/java/io/quarkus/undertow/test/builtinbeans/ServletBuiltinBeanInjectingBean.java

```diff
@@ -0,0 +1,37 @@
+package io.quarkus.undertow.test.builtinbeans;
+
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+import javax.servlet.ServletContext;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+
+@Dependent
+class ServletBuiltinBeanInjectingBean {
+
+    @Inject
+    HttpServletRequest request;
+
+    @Inject
+    HttpSession session;
+
+    @Inject
+    ServletContext context;
+
+    public void verifyRequest() {
+        assert request.getRequestURI().contains("/request");
+        assert request.getParameter("foo").equals("bar");
+    }
+
+    public void verifySession() {
+        assert session.isNew();
```

Comment:
I'm not 100% sure we always run the tests with `-enableassertions`. It's probably safer to use `org.junit.jupiter.api.Assertions`...

Reply:
Got it. I'll update the test case to use `org.junit.jupiter.api.Assertions` API.